src/pds/registrysweepers/legacy_registry_sync/legacy_registry_sync.py

```diff
@@ -0,0 +1,64 @@
+from typing import Union
+from solr_to_es.solrSource import SlowSolrDocs
+from elasticsearch import Elasticsearch
+import elasticsearch.helpers
+import logging
+from pds.registrysweepers.utils import configure_logging
+from pds.registrysweepers.legacy_registry_sync.solr_doc_export_to_opensearch import SolrOsWrapperIter
+from pds.registrysweepers.legacy_registry_sync.opensearch_loaded_product import get_already_loaded_lidvids
+
+log = logging.getLogger(__name__)
+
+SOLR_URL = 'https://pds.nasa.gov/services/search/search'
+OS_INDEX = "legacy_registry"
+
+
+def create_legacy_registry_index(es_conn=None):
+    if not es_conn.indices.exists(OS_INDEX):
+        log.info("create index %s", OS_INDEX)
+        _ = es_conn.indices.create(
```

Comment:
Change ` _ = es_conn.indices.create(` → `es_conn.indices.create(` (since the return value isn't used).

Reply:
yes, silly me